djstripe/views.py

```diff
@@ -49,7 +49,7 @@ def get_context_data(self, *args, **kwargs):
         context['customer'] = customer
         try:
             context['subscription'] = customer.current_subscription
```

Comment:
@pydanny, what do you think about the current_subscription backwards compatability? I'd prefer to drop it, but things could get a bit messy.